src/plugins/thirdroom/thirdroom.game.ts

```diff
@@ -196,6 +219,18 @@ export const ThirdRoomModule = defineModule<GameState, ThirdRoomModuleState>({
       create: createAvatarRig(input, physics),
     });
 
+    registerPrefab(ctx, {
```

Comment:
I'm not sure if you meant to add this to this PR? Probably best to include it in the AR avatar PR you're working on next so that everything can be reviewed at the same time.

Reply:
whoops, these indeed slipped through lol